custom-recipes/amazon-transcribe/recipe.json

```diff
@@ -35,6 +35,15 @@
             "required": true,
             "acceptsDataset": true,
             "acceptsManagedFolder": false
+        },
+        {
+            "name": "output_folder",
+            "label": "Output managed folder",
+            "description": "Managed folder connected to S3 bucket that contains the raw results from the AWS jobs.",
```

Comment:
I think it's worth mentioning that it's optional in the description (it's a pity that we don't take into account `required` flag in the UI)